Tests-Legacy/L0/Maven/_suite.ts

```diff
@@ -549,6 +549,35 @@ describe('Maven Suite', function () {
             });
     })
 
+    it('run maven with feed', (done) => {
```

Comment:
Will be good to add a test for "run maven with feed when user specifies settings file" as well.